robotidy/app.py

```diff
@@ -100,8 +100,8 @@ def save_model(self, model):
     def output_diff(self, path: str, old_model: StatementLinesCollector, new_model: StatementLinesCollector):
         if not self.show_diff:
             return
-        old = old_model.text.splitlines()
-        new = new_model.text.splitlines()
+        old = [l + '\n' for l in old_model.text.splitlines() if l]
```

Comment:
The PR looks good to be merged - if you take care of that `if l` I will approve it

Reply:
(To be honest I did not fully test it with other transformers but I assume it's okay since it's extra option - we can add tests & fix issues over the time)

Reply:
Thanks for taking care of that change, and merging this!